src/pom.xml

```diff
@@ -119,17 +120,18 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <webapp-archetype-resource-pom>shogun-core-webapp-archetype/src/main/resources/archetype-resources/pom.xml</webapp-archetype-resource-pom>
+        <webapp-archetype-resource-pom>shogun-core-webapp-archetype/src/main/resources/archetype-resources/pom.xml
+        </webapp-archetype-resource-pom>
 
         <tomcat.version>9.0.30</tomcat.version>
 
         <!-- Core -->
-        <spring.version>5.2.3.RELEASE</spring.version>
-        <spring-security.version>5.2.2.RELEASE</spring-security.version>
+        <spring.version>5.3.10</spring.version>
+        <spring-security.version>5.5.2</spring-security.version>
         <spring-ldap.version>2.3.2.RELEASE</spring-ldap.version>
-        <log4j.version>2.13.3</log4j.version>
+        <log4j.version>2.14.1</log4j.version>
         <slf4j.version>1.7.28</slf4j.version>
-        <jackson.version>2.10.2</jackson.version>
+        <jackson.version>2.13.0-rc2</jackson.version>
```

Comment:
Is the rc version required? If not I would prefer to stick to version 2.12.

Reply:
No, not required. But I also upgraded to this rc version in a couple of other projects and also had no problems there. Should I still downgrade?

Reply:
If you didn't encounter any problems, it's absolutely fine for me.